test/units/cli/test_galaxy.py

```diff
@@ -444,3 +460,475 @@ def test_template_ignore_similar_folder(self):
 
     def test_skeleton_option(self):
         self.assertEquals(self.role_skeleton_path, context.CLIARGS['role_skeleton'], msg='Skeleton path was not parsed properly from the command line')
+
+
+@pytest.fixture()
+def collection_skeleton(request, tmp_path_factory):
+    name, skeleton_path = request.param
+
+    galaxy_args = ['ansible-galaxy', 'collection', 'init', '-c']
+
+    if skeleton_path is not None:
+        galaxy_args += ['--collection-skeleton', skeleton_path]
+
+    test_dir = to_text(tmp_path_factory.mktemp('test-ÅÑŚÌβŁÈ Collections'))
```

Comment:
I think it's a good practice to use explicit unicode string literals in places like this
```suggestion
    test_dir = to_text(tmp_path_factory.mktemp(u'test-ÅÑŚÌβŁÈ Collections'))
```

Reply:
tmp_path_factory fails on Python 2 for unicode strings as it just conditionally tries to encode it as ASCII.